include/swift/SILOptimizer/Utils/Local.h

```diff
@@ -79,7 +79,8 @@ bool isIntermediateRelease(SILInstruction *I,
 
 /// \brief Recursively collect all the uses and transistive uses of the
 /// instruction.
-void collectUsesOfValue(SILValue V, llvm::DenseSet<SILInstruction *> &Insts);
+void
+collectUsesOfValue(SILValue V, llvm::SmallPtrSet<SILInstruction *, 4> &Insts);
```

Comment:
IIRC there is a SmallPtrSetImpl. I think you can use it like llvm::SmallVectorImpl and thus avoid the need to put in the size here.

But my memory may be incorrect.